crates/vault-registry/src/tests.rs

```diff
@@ -720,7 +720,7 @@ fn liquidate_at_most_secure_threshold() {
         create_vault_with_collateral(&vault_id, backing_collateral);
         let liquidation_vault_before = VaultRegistry::get_rich_liquidation_vault(&DEFAULT_CURRENCY_PAIR);
 
-        VaultRegistry::_set_secure_collateral_threshold(
+        VaultRegistry::_set_liquidation_collateral_threshold(
```

Comment:
Not sure what the default secure threshold is, but let's explicitly set premium-redeem and secure thresholds to 175 and 200% respectively so we're sure the test is not passing only because the thresholds are the same